docs/reference/admin/layouts/layout-kit.mdx

```diff
@@ -0,0 +1,199 @@
+Its main purpose is to manage state of panels by wrapping the `LayoutPrimitives.ResponsiveContainer`.
+The responsive container provides means for `LayoutKit.ContentMainPanel`, `LayoutKit.SidebarLeft`
+and `LayoutKit.SidebarRight` to work seamlessly together without much effort on your side.
+
+Together with toggle buttons your have a fully functional layout with responsive panels you can
+use to compose most application layouts on top of.
+
+## `LayoutKit.Frame`
+
+`LayoutKit.Frame` is a container component that provides a basic frame for your app,
+it accepts `header` and `footer` bars, or `children` via props. Its purpose is to manage
+state of panels but it also calculates safe area insets for panels added as its children,
+e.g. when header or footer are in fixed positions.
+
+:::tip Handling "safe-areas" of your app
+You will need to wrap your layout component	with `<SafeAreaInsetsProvider>` component
+but also update `viewport` meta tag of `index.html` by adding `viewport-fit=cover` value
+by which you instruct browser that you will handle the safe area insets yourself:
+
+```html
+<meta name="viewport" content="width=device-width, viewport-fit=cover, initial-scale=1">
+```
+:::
+
+```tsx title="admin/components/LayoutComponent.tsx"
+import { LayoutKit, SafeAreaInsetsProvider } from '@contember/layout'
+import { PropsWithChildren } from 'react'
+import '@contember/layout/index.css'
+
+export function LayoutComponent({ children }: PropsWithChildren) {
+	return (
+		<SafeAreaInsetsProvider>
+			<LayoutKit.Frame
+				header={<h1>Welcome!</h1>}
+				children={children}
+				footer={<p><small>Created with <a className="content-link" href="https://www.contember.com/">AI-assisted Contember Studio</a></small></p>}
+			/>
+		</SafeAreaInsetsProvider>
+	)
+}
+```
+
+## `LayoutKit.Header` and `LayoutKit.Footer`
+
+`LayoutKit.Header` and `LayoutKit.Footer` bars are pre-defined components that provide 3 areas for content – `start`, `center` and `end`.
+
+```tsx title="admin/components/LayoutComponent.tsx"
+import { LayoutKit, SafeAreaInsetsProvider } from '@contember/layout'
+import { PropsWithChildren } from 'react'
+import '@contember/layout/index.css'
+
+export function LayoutComponent({ children }: PropsWithChildren) {
+	return (
+		<SafeAreaInsetsProvider>
+			<LayoutKit.Frame
+				header={
+					<LayoutKit.Header
+						start={<a href="#">Back</a>}
+						center={<h1>Contember</h1>}
+						end={<a href="#">Logout</a>}
+					/>
+				}
+				children={children}
+				footer={
+					<LayoutKit.Footer
+						start={false}
+						center={<p><small>Created with <a className="content-link" href="https://www.contember.com/">AI-assisted Contember Studio</a></small></p>}
+						end={false}
+					/>
+				}
+			/>
+		</SafeAreaInsetsProvider>
+	)
+}
+```
+
+:::tip When you pass `false` to `start`, `center` or `end` props, `<div>` wrapper around the area won't be rendered.
+:::
+
+
+You can also create own instance of bar component:
```

Comment:
what is a purpose of creating custom bar component?

Reply:
ok, add it to the docs

Reply:
Done